R/derive_var_dthcaus.R

```diff
@@ -2,7 +2,7 @@
 #'
 #' Derive death cause (`DTHCAUS`) and add traceability variables if required.
 #'
-#' @param dataset Input dataset. `USUBJID` is an expected column.
+#' @param dataset_name Input dataset. `USUBJID` is an expected column.
```

Comment:
Please add the `source_datasets` parameter. You can use a similar description as in `derive_param_tte()`.